### ISSUE TITLE
[Egg Request]: Sonic robo blast 2 (SRB2)

### ISSUE THREAD

**Issue by fizztune** [parkervcp/eggs#1300]
### Service

srb2

### Does this expand an already existing service

Yes

### Link to game

https://www.srb2.org/

### Links for server downloads

You need to compile it on linux - https://wiki.srb2.org/wiki/Source_code_compiling

### Links for the install docs

Run it with -dedicated command line argument. Allow optional "Master Server ID" with command line argument of -room, which must be 33, 28, 31, 38

**Comment by tmunsch** [comment]
I've tried this for about an hour and have been unsuccessful. It require flatpak which seems to go through the install but becomes unavailable afterwards. I will try the compile from source method tomorrow.

**Comment by tmunsch** [comment]
I don't think this will be viable. The compiling of the source in tow is too labor intensive and would make a server install take 20-30 minutes minimum. Plus the compiling didn't complete and was missing some vital .pk3 files. I had to manually move them in order to get the server to run. If they actually provided a linux build this wouldn't be an issue.

**Comment by fizztune** [comment]
> I don't think this will be viable. The compiling of the source in tow is too labor intensive and would make a server install take 20-30 minutes minimum. Plus the compiling didn't complete and was missing some vital .pk3 files. I had to manually move them in order to get the server to run. If they actually provided a linux build this wouldn't be an issue.

I had the same problem, I had to put the pk3 files in it. We could just use a precompiled build, since it is licensed under GNU General Public License v2.0.
Another option would be asking for a zip file to upload and run, users would have to compile themselves though.

**Comment by parkervcp** [comment]
I am pretty sure I have a working script to build and run the server. I'll have a PR out soon.

**Comment by fizztune** [comment]
@parkervcp Sorry for mentioning you, but could we please have an update? thanks

**Comment by gOOvER** [comment]
> @parkervcp Sorry for mentioning you, but could we please have an update? thanks

It's done, when it's done. No need to ask for it.
